ipfs-api/Cargo.toml

```diff
@@ -15,34 +15,39 @@ license                   = "MIT OR Apache-2.0"
 travis-ci                 = { repository = "ferristseng/rust-ipfs-api" }
 
 [features]
-default                   = ["with-hyper-tls", "with-builder"]
+default                   = ["with-actix"]
```

Comment:
Odd to add reqwest but change the default to actix? ;)

Reply:
good catch! I'll change that.